markdown/chapters/meandeck/basics.md

```diff
@@ -1,334 +1,186 @@
-two cards left in your library on its resolution, other times it means needing
-to have no cards left in your library on resolution. The former is known as an
-_imperfect pile_ and the latter a _perfect pile_. An _imperfect pile_ is named
-as such because if your opponent is able to remove the Oracle from play prior to
-resolution of its trigger then you will not win the game.
+Welcome to Chapter 0 of the Doomsday section of the Wiki.
+
+This is the start of your journey into trying to learn a bit more about the deck
+and hopefully will inspire and help you to progress further with it!
+
+## We're Teaching People to Fish
+
+As the Wiki front page states:
+
+> This is specifically not a _primer_ as you often see for other decks in
+> Legacy, because primers generally become outdated rather quickly and are more
+> about card choices, decklists, sideboard plans, and so on. The Doomsday Wiki
+> is there to help people to really understand how the deck works and can win in
+> various scenarios.
+
+As a result a lot of the information presented within these chapters is to be
+taken as guidance only. The format (and the deck) changes as new cards are
+printed or existing ones banned or unbanned and, with the speed with which
+conversation and innovation can occur, vastly outpaces the collective authors'
+ability to keep this site up to date. We try to keep things as high level and
+universally applicable as possible. As a result do not seek to try and rote
+memorise any _piles_ presented here but instead try and use them to ingrain core
+concepts or ideas into your play which you can then supplement with experience
+as you practice. There are plenty of people out there who will happily share
+tables of _piles_ which you can use however it might be worth taking a few
+losses early on whilst learning the deck to understand what you are doing rather
+than winning but getting railroaded in only a few scenarios.
+
+## There are No Stupid Questions and Very Few Original Ones
+
+Questions are good. Quizzing things helps learning and understanding. If you
+don't start making inquiries into why things are as they are, then you're never
+going to get the most out of any deck you play. Conversely though, a lot of the
+questions you might want to ask are very likely to have been asked before. This
+is great as it means it is hopefully written down somewhere already! If you
+think of something you want answered and cannot find it in the chapters there
+are a few places you can look which might help. First and foremost is the
+[FAQs](/appendices/faq)! Otherwise, articles are dated content which are more
+likely to have recent changes or information inside them.
+
+The next port of call is other web sources like the MtGLegacy Reddit or
+Similar Forums. These sometimes have players ask questions. Use the search
+function within the subreddit for the word "Doomsday" which may help you find
+the answer. Very likely though the best place to search after the Wiki is the
```

Comment:
I doubt /r/mtglegacy is the first place to recommend as there's often misinformation and the search feature is probably not as good as Discord's

Wdyt?